src/components/MeetingTypes/MeetingTypeCard.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+
+import { Card, Button, Tag, Typography, notification, Icon } from "antd";
+import "./style.scss";
+
+import EditMeetingTypeForm from "./EditMeetingTypeForm";
+import { connect } from "react-redux";
+
+class MeetingTypeCard extends Component {
+  state = {
+    editing: false,
+    loading: false,
+  };
+
+  handleFormOpen = () => {
+    this.setState({ editing: !this.state.editing });
+  };
+
+  // handleFormSubmit = (formValues) => {
```

Comment:
delete?

Reply:
I've deleted the commented code. I'm assuming that's what you mean.

Reply:
yeah